app/views/catalog/_cited_documents_default.html.erb

```diff
@@ -1,7 +1,7 @@
 <% if @document.reference? && @document.cites_other_documents? %>
   <div class='col-md-12 record-metadata-section'
        data-behavior='cited-documents-contents'
-       data-path="<%= search_exhibit_catalog_path(exhibit_id: @exhibit) %>"
+       data-path="<%= url_for([@spotlight, :documents_list]) %>"
```

Comment:
What's the purpose of `@spotlight` here? I think `url_for(:documents_list)` ought to work.

Reply:
Correct, I changed it